CHANGELOG.md

```diff
@@ -84,6 +84,7 @@ if the provided arguments are invalid.
 * (x/auth) [\#5006](https://github.com/cosmos/cosmos-sdk/pull/5006) The gas required to pass the `AnteHandler` has
 increased significantly due to modular `AnteHandler` support. Increase GasLimit accordingly.
 * (rest) [\#5336](https://github.com/cosmos/cosmos-sdk/issues/5336) `MsgEditValidator` uses `description` instead of `Description` as a JSON key.
+* (keys) [\#5097](https://github.com/cosmos/cosmos-sdk/pull/5097) Due to the keybase -> keyring transition, keys need to be migrated. See `keys migrate` command for more info.
```

Comment:
is this entry correct?

Reply:
I think so

Reply:
the PR number is incorrect...

Reply:
OK. This new changelog entry though should have been introduced in #5097 and is not about `--keyring-file`. I'm essentially pushing a bugfix to the old PR with this change :)